frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -399,122 +404,134 @@ export const ProjectFeatureToggles = ({
-                        onChange={setTableState}
-                        state={filterState}
-                    />
-                    <SearchHighlightProvider value={tableState.query || ''}>
-                        <PaginatedTable
-                            tableInstance={table}
-                            totalItems={total}
-                        />
-                    </SearchHighlightProvider>
-                    <ConditionallyRender
-                        condition={!data.length && !isPlaceholder}
-                        show={
-                            <TableEmptyState query={tableState.query || ''} />
-                        }
-                    />
-                    {rowActionsDialogs}
-
-                    {featureToggleModals}
+                        bodyClass='noop'
+                        style={{ cursor: 'inherit' }}
+                    >
+                        <div
+                            ref={bodyLoadingRef}
+                            aria-busy={isPlaceholder}
+                            aria-live='polite'
+                        >
+                            <ProjectOverviewFilters
+                                project={projectId}
+                                onChange={setTableState}
+                                state={filterState}
+                            />
+                            <SearchHighlightProvider
+                                value={tableState.query || ''}
+                            >
+                                <PaginatedTable
+                                    tableInstance={table}
+                                    totalItems={total}
+                                />
+                            </SearchHighlightProvider>
+                            <ConditionallyRender
+                                condition={!data.length && !isPlaceholder}
+                                show={
+                                    <TableEmptyState
+                                        query={tableState.query || ''}
+                                    />
+                                }
+                            />
+                            {rowActionsDialogs}
 
-                    {'feature' in project.onboardingStatus ? (
-                        <ConnectSdkDialog
-                            open={connectSdkOpen}
-                            onClose={() => {
-                                setConnectSdkOpen(false);
-                            }}
-                            project={projectId}
-                            environments={environments}
-                            feature={project.onboardingStatus.feature}
-                        />
-                    ) : null}
-                </div>
-            </PageContent>
+                            {featureToggleModals}
+                        </div>
+                    </PageContent>
+                }
+            />
+            {'feature' in project.onboardingStatus ? (
```

Comment:
@kwasniew moved this out of feature toggle list, because if we hide the list, dialog can not open